frontend/src/components/menu/Menu.styled.ts

```diff
@@ -67,6 +67,11 @@ const ChevronIcon = styled(BsChevronDown)`
 const DropdownWrapper = styled.div`
     position: relative;
     top: 2rem;
+
+    @media (max-width: 800px) {
```

Comment:
Create a value as a const, because it's easy to manage when u change a break point.

Good practice is use break points from const, because when u change whole design to other size/colors etc.

And u have etc. 3 break point, u can easly avoid mistakes, situation when u have a several components which brakes in 800px, and 3 which break in 600 px or STM.

Reply:
all media queries and color constans are now stored and referenced via "./assets/styles/theme"